vllm/model_executor/models/utils.py

```diff
@@ -402,9 +402,17 @@ def merge_multimodal_embeddings(
     Note:
         This updates ``inputs_embeds`` in place.
     """
+    if isinstance(placeholder_token_id, int):
+        return _merge_multimodal_embeddings(
+            inputs_embeds,
+            (input_ids in placeholder_token_id),
```

Comment:
```suggestion
            (input_ids == placeholder_token_id),
```

I think you made a typo here?

Reply:
Ah yes!